dcpy/lifecycle/package/validate.py

```diff
@@ -103,12 +110,14 @@ def _is_geom_poly(s):
 
 col_validators = {
     "bbl": lambda df, col_name: df[~df[col_name].str.match(r"^\d{10}$")],
-    "integer": lambda df, col_name: df[~df[col_name].apply(_is_int)],
-    "double": lambda df, col_name: df[~df[col_name].apply(_is_float_or_double)],
     "wkb": lambda df, col_name: df[~df[col_name].apply(_is_valid_wkb)],
```

Comment:
We should probably remove the geometry types - leaving that til we've figured out what we actually want to do.